recipe-server/client/control_new/components/extensions/CreateExtensionPage.js

```diff
@@ -0,0 +1,57 @@
+import { message } from 'antd';
+import autobind from 'autobind-decorator';
+import React, { PropTypes as pt } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'redux-little-router';
+
+import ExtensionForm from 'control_new/components/extensions/ExtensionForm';
+import { createExtension } from 'control_new/state/extensions/actions';
+
+
+@autobind
+export class _CreateExtensionPage extends React.Component {
```

Comment:
I swear this used to be the recommended style in ant-design's docs, but looking again, they name the wrapped component `WrappedThing` instead.

I went with what I thought was their style because having a default import named `CreateExtensionPage` as well as a member import named `CreateExtensionPage` that are in fact two different things is confusing; if you import both in, say, a test, you end up naming the default import something different than what the rest of the app calls it. Having a special name for the unwrapped non-default export gets around this.

I'm not partial to the underscore; `UnwrappedCreateExtensionPage`, `UnconnectedCreateExtensionPage`, or whatever else you or @rehandalal want is equally fine.